### ISSUE TITLE
Link to 'Open in a new tab' broken for Adapter Readme

### ISSUE THREAD

**Issue by Grothesk242** [ioBroker/ioBroker.admin#2032]
admin 6.7.0
nodejs 20.5.0 
js-controller 5.0.11

Trying to open the adapter readme in a new tab does nothing. 
In Brower Konsole it says: 

```
Cannot semver: TypeError: Invalid Version: 2.5.0/1 [Markdown.jsx:1000:20](http://chet.fritz.box:8081/static/js/components/Markdown.jsx)
Some cookies are misusing the recommended “SameSite“ attribute 6
Uncaught uncaught exception: [object PromiseRejectionEvent] [bootstrap.jsx:100:8](http://chet.fritz.box:8081/static/js/bootstrap.jsx)
Uncaught 
unhandledrejection { target: Window, isTrusted: true, promise: Promise { "rejected" }, reason: "timeout", srcElement: Window, eventPhase: 0, bubbles: false, cancelable: true, returnValue: true, defaultPrevented: false, … }
[bootstrap.jsx:113:8](http://chet.fritz.box:8081/static/js/bootstrap.jsx)
Uncaught (in promise) timeout
Uncaught TypeError: e.openTab is not a function [bootstrap.jsx:100:8](http://chet.fritz.box:8081/static/js/bootstrap.jsx)
Uncaught TypeError: e.openTab is not a function
    onClick AdapterInfoDialog.jsx:246
    React 11
```

**Comment by Feuer-sturm** [comment]
Perhaps the fix of #2003 causes this issue @foxriver76

**Comment by foxriver76** [comment]
No probably there is more broken. BF removed some code which seemed to be unused and refactored some which lead to some issues as seen in the last versions.
